examples/stable-diffusion/training/train_text_to_image_sdxl.py

```diff
@@ -1406,7 +1428,14 @@ def compute_time_ids(original_size, crops_coords_top_left):
                 if args.prediction_type is not None:
```

Comment:
Why cant we just assign scheduler here in pipeline = ( ... directly? 
Like:
```
pipeline = GaudiStableDiffusionXLPipeline.from_pretrained(
    args.pretrained_model_name_or_path,
    scheduler=noise_schedler, 
    ...               
```
Since we already went through selection trouble earlier for `noise_scheduler` object.
Then get rid of all added code below :)

Only keep:
```
if args.prediction_type is not None:
    scheduler_args = {"prediction_type": args.prediction_type}
    pipeline.scheduler = pipeline.scheduler.from_config(pipeline.scheduler.config, **scheduler_args)
```

Reply:
ok.
Just wonder original code has hardcore here.

Reply:
Add `scheduler=noise_scheduler` in `pipeline = GaudiStableDiffusionXLPipeline.from_pretrained(` above

Reply:
The `pipeline` in this block will be delete, as well as `noise_scheduler`, so use `pipeline.scheduler.from_config(noise_scheduler.config)`